core/domain/customization_args_util.py

```diff
@@ -118,8 +120,9 @@ def validate_customization_args_and_values(
                 schema_utils.normalize_against_schema(
                     customization_args[ca_spec.name]['value'],
                     ca_spec.schema))
-        except Exception:
+        except Exception as e:
```

Comment:
Should we only except validation error here? (not sure!)

Reply:
Yes, makes sense. Done.

Reply:
Not done?

Reply:
My bad - I confused it with the jobs file. We cannot except only utils.ValidationError here since schema_utils raises normal Exception as well as AssertionErrors. Further, I reverted the changes in job file to except general exceptions as well to handle any unexpected errors during the run.